src/cli/README.md

```diff
@@ -358,12 +358,99 @@ Done
 
 Print border agent state.
 
+Possible states are
+
+- `Stopped` : Border Agent is stopped.
+- `Started` : Border Agent is running with no active connection with external commissioner.
+- `Active` : Border Agent is running and is connected with an external commissioner.
+
 ```bash
 > ba state
 Started
 Done
 ```
 
+### ba ephemeralkey
+
+Indicates if an ephemeral key is in-use by the Border Agent.
+
+Requires `OPENTHREAD_CONFIG_BORDER_AGENT_EPHEMERAL_KEY_ENABLE`.
+
+```bash
+> ba ephemeralkey
+not in-use
```

Comment:
How about `active` and `inactive`?

Reply:
Good idea. Thanks. Changed this in new pushed commit.